main.go

```diff
@@ -634,16 +634,15 @@ func main() {
 			usage()
 			os.Exit(1)
 		}
-		if flag.NArg() != 1 {
-			fmt.Fprintln(os.Stderr, "No package specified.")
-			usage()
-			os.Exit(1)
+		pkgName := "."
+		if flag.NArg() == 1 {
```

Comment:
So when you pass more than one parameter, it still defaults to the current directory? I think we should print an error when there is more than one positional parameter specified. (The same goes for the other two commands below).

Reply:
That seems like a good idea.